examples/recording_service/leveldb-integration/cpp/README.md

```diff
@@ -0,0 +1,1077 @@
+# Recording Service -- Integration with LevelDB
+
+## Example Description
+
+This example is an implementation of the pluggable *RTI Recording Service* 
+storage, using the no-SQL, persistent key-value store, LevelDB. This engine was
+developed by Google as a fast key-value storage library that maps from key 
+strings to string values. The LevelDB code can be found on 
+[GitHub](https://github.com/google/leveldb).
+
+This example is written in modern C++ (C++11) and shows how integration with a
+no-SQL storage technology can be done in *RTI Recording Service*. The example
+implements both the Storage Writer and Storage Reader plug-ins, so it can be
+used to record, replay and/or convert the data coming from the Connext DDS data
+bus. However, take into account that LevelDB is not meant to be accessed by two
+different processes at the same time, and hence recording and replaying (or
+converting) concurrently is _not supported_. In general, concurrently accessing
+the LevelDB database being recorded, replayed or converted is not supported in 
+this example. Note that this is not the case with our built-in SQLite 
+implementation, although concurrent access in that case is also discouraged.
+
+This example will store any DDS Topic discovered by *RTI Recording Service* that
+has been marked as to be recorded (by defining the Topics and Topic Groups in 
+the XML configuration).
+
+The example provides the following files:
```

Comment:
All the content up to this point should be in a README.rmd that goes to the top-level directory (leveldb-integration). Then under cpp, we just leave the readme with specific content for this language, like the files description and build instructions

Reply:
Will do last thing.